adafruit_epd/ssd1680.py

```diff
@@ -211,3 +211,52 @@ def set_ram_address(
         self.command(_SSD1680_SET_RAMXCOUNT, bytearray([x + 1]))
         # Set RAM Y address counter
         self.command(_SSD1680_SET_RAMYCOUNT, bytearray([y, y >> 8]))
+
+
+class Adafruit_SSD1680Z(Adafruit_SSD1680):
+    """Driver for SSD1680Z ePaper display, overriding SSD1680 settings."""
+
+    # pylint: disable=too-many-arguments, useless-parent-delegation
+    def __init__(
+        self, width, height, spi, *, cs_pin, dc_pin, sramcs_pin, rst_pin, busy_pin
+    ):
+        # Call the parent class's __init__() to initialize attributes
+        super().__init__(
+            width,
+            height,
+            spi,
+            cs_pin=cs_pin,
+            dc_pin=dc_pin,
+            sramcs_pin=sramcs_pin,
+            rst_pin=rst_pin,
+            busy_pin=busy_pin,
+        )
+        self.busy_pin = busy_pin  # Ensure busy_pin is set
+
+    # pylint: enable=too-many-arguments, useless-parent-delegation
+
+    def power_up(self):
+        """Power up sequence specifically for SSD1680Z."""
+        self.hardware_reset()
+        self.busy_wait()
+        self.command(_SSD1680_SW_RESET)
+        self.busy_wait()
+
+        self.command(
+            _SSD1680_DRIVER_CONTROL,
+            bytearray([self._height - 1, (self._height - 1) >> 8, 0x00]),
+        )
+        self.command(_SSD1680_DATA_MODE, bytearray([0x03]))
```

Comment:
If I'm understanding the chart in the PR description correctly, shouldn't this be `0x01`?



> Feature/Command | SSD1680 Library     | SSD1680Z Library
> ----------------|---------------------|-----------------
> Data Entry Mode | `bytearray([0x03])` | `bytearray([0x01])`
> ...             | ...                 | ...

I'm not familiar with these chips (yet), so I'm still trying to figure out where the old vs new values came from. If you could share any pointers regarding where/how you found the new values, it would be much appreciated!